specs/deneb/beacon-chain.md

```diff
@@ -266,6 +268,29 @@ def process_execution_payload(state: BeaconState, body: BeaconBlockBody, executi
     )
 ```
 
+#### Modified `process_voluntary_exit`
```

Comment:
We have a precedent of doing these type of note call outs to help the reader see the change
```suggestion
#### Modified `process_voluntary_exit`

Note: The function `process_voluntary_exit` is modified to use the a fixed fork version -- `CAPELLA_FORK_VERSION` -- for EIP-7044
```